appinventor/appengine/src/com/google/appinventor/server/CrashReport.java

```diff
@@ -49,6 +49,14 @@ public static RuntimeException createAndLogError(Logger log, HttpServletRequest
       new RuntimeException(exception);
   }
 
+  public static RuntimeException createAndLogError(org.apache.log4j.Logger log, HttpServletRequest req,
```

Comment:
I added a Javadoc. It is not used here but is used in the Stand Alone and Scale-able code (where I am using log4j). This code was carried over as an artifact of how I removed the gallery, to be consistent with how I removed it a while ago from the stand alone and scale-able code. Leaving it here makes the merge of this code into the standalone and scale-able code easier.

Reply:
It might be better if we can either use the Java Logging framework or the Apache framework. I don't really see the point of mixing the two frameworks.